cypress/integration/navigations.ts

```diff
@@ -0,0 +1,58 @@
+import { IFRAME_SELECTOR } from '../support/constants';
+
+describe('navigations', () => {
```

Comment:
I would give this a more descriptive name like "navigating an EPUB" or something